babel.config.cjs

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: [['@babel/preset-env', {targets: {node: 'current'}}]],
+};
```

Comment:
you could expand this out so it's easier to read, but you don't have to. I think you have an extra set of `[]`

```js
module.exports = {
  presets: [
    [
    '@babel/preset-env',
    {
      targets: {
        node: 'current'
      }
    }
   ]
  ],
};